src/attach.js

```diff
@@ -16,8 +16,19 @@ function interceptor(fetch, ...args) {
     }
   });
 
+  let request = null;
```

Comment:
This will not work cause what do you do when you have multiple requests? Your `request` variable will get overwritten by other concurrent requests.